get_started.md

```diff
@@ -2,42 +2,59 @@
 
 The Gaia Sandbox provides a containerized simulation environment that allows you to experiment with samples written using the Gaia Platform.
 
-This sample models a factory that contains the following:
+This sample models a security access control to allow or deny people entering an office campus. It contains:
 
-TODO
+- A headquarters (HQ) building
+- Rooms inside the HQ building
+- People, of varying security access levels
```

Comment:
is security needed here? reads oddly to me.
People with different levels of access to the facilities:
*End this with a colon)